client/views/helpers/nav/nav.scss

```diff
@@ -151,6 +151,18 @@ $screen-sm-max: ($screen-md - 1);
 	}
 }
 
+#navArchive {
+	background-color: #4286f4;
+
+	&:hover {
+		background-color: #88adea;
+	}
+
+	.fa {
+		font-size: 1em !important;
```

Comment:
@singhsanket143 can you try to not to use `!important`? Generally there is always a better way to do it.